openmc/tallies.py

```diff
@@ -1105,7 +1116,7 @@ def get_values(self, scores=[], filters=[], filter_bins=[],
                   '\'rel_err\', \'sum\', or \'sum_sq\''.format(self.id, value)
             raise LookupError(msg)
 
-        return data
+        return data.copy()
```

Comment:
It looked to me like this was performing basic indexing of the array and returning a view instead of performing advanced indexing and returning a copy. Upon testing it seems this is performing advanced indexing and thus returning a copy of the values. I'll change this back to `return data`